src/lib/components/layout/Sidebar/UserMenu.svelte

```
@@ -5,10 +5,20 @@
 	import { flyAndScale } from '$lib/utils/transitions';
 	import { goto } from '$app/navigation';
 	import ArchiveBox from '$lib/components/icons/ArchiveBox.svelte';
-	import { showSettings, activeUserIds, USAGE_POOL, mobile, showSidebar } from '$lib/stores';
+	import {
+		showSettings,
```

ESLint

[error] 'showSettings' is defined but never used.

(@typescript-eslint/no-unused-vars)